doc/classes/PhysicsDirectBodyState2D.xml

```diff
@@ -76,6 +76,7 @@
 			<param index="0" name="torque" type="float" />
 			<description>
 				Applies a rotational force without affecting position. A force is time dependent and meant to be applied every physics update.
+				[b]Note:[/b] [member inverse_inertia] is required for this to work.  To have [member inverse_inertia], an active [CollisionShape2D] must be a child of the node, or you can manually set [member inverse_inertia].
```

Comment:
We only use one space after a period in the docs. (Same for all added sentences.)